### DISCUSSION TITLE
SFML

### DISCUSSION THREAD

**Discussion by LouisDodorico** [Homebrew/discussions#3738] [Getting started]
I have just downloaded brew and am trying to use sfml for a uni project. Two of my friends have just used brew to download sfml but when I search for sfml in terminal it says:
Error: No formulae or casks found for "sfml".

**Comment by gromgit** [comment]
What's the output of:
```
brew config
brew doctor
```
?

**Reply by LouisDodorico** [reply]
staff-214-185:~ louisdodorico$ brew config
brew doctor
HOMEBREW_VERSION: 3.6.4-33-gc49f2e8
ORIGIN: https://github.com/Homebrew/brew
HEAD: c49f2e8b4c343844130b7517c84c3b1bec16a332
Last commit: 17 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: c49f2e8b4c343844130b7517c84c3b1bec16a332
Core tap last commit: 17 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 13.1.6 build 1316
Git: 2.32.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6-x86_64
CLT: 13.4.0.0.1.1651278267
Xcode: N/A
staff-214-185:~ louisdodorico$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core
staff-214-185:~ louisdodorico$

**Reply by LouisDodorico** [reply]
i have just run what it said to run, so ill see if that fixes it

**Reply by LouisDodorico** [reply]
all fixed, thanks for the help
